python/cudf_polars/cudf_polars/testing/plugin.py

```diff
@@ -45,6 +45,7 @@ def pytest_configure(config: pytest.Config) -> None:
 
 
 EXPECTED_FAILURES: Mapping[str, str] = {
+    "tests/unit/dataframe/test_df.py::test_extension": "AssertionError",
```

Comment:
I could use some help understanding why this test is failing. It fails at the first ref count assertion. The test is here https://github.com/pola-rs/polars/blob/40f8f5d63c225cd3dcb4e220db1bd5622274b2ab/py-polars/tests/unit/dataframe/test_df.py#L1820

Reply:
Does it fail for you on branch-24.12, I guess not?

Reply:
I think so. I'm going to let it run again and see what I can figure out